packages/trip-form/src/test-utils/modes.js

```diff
@@ -1,20 +1,110 @@
 const commonModes = {
   transitModes: [
     {
+      image: "https://trimet.org/global/img/mode-bus.png",
       mode: "BUS",
-      label: "Bus"
+      label: "Buses"
     },
     {
+      image: "https://trimet.org/global/img/mode-max.png",
       mode: "TRAM",
-      label: "MAX & Streetcar"
+      label: "MAX and Portland Streetcar"
     },
     {
+      image: "https://trimet.org/global/img/mode-wes.png",
       mode: "RAIL",
-      label: "WES"
+      label: "WES Commuter Rail"
     },
     {
       mode: "GONDOLA",
-      label: "Aerial Tram"
+      label: "Aerial Tram",
+      hidden: true
+    }
+  ],
+  categories: [
+    {
+      description:
+        "E-scooters are battery-powered devices that you can rent on-the-spot using an app on your phone. You can quickly and easily connect to and from your transit stop, using one of several scooter brands operating in the Portland area.",
+      id: "BICYCLE",
+      image:
+        "https://cdn.shopify.com/s/files/1/0514/3160/4395/files/ronin-titanium-natural-sram-red-axs_1024x.png?v=1617366397",
+      label: `+ Bike or Bike${"\u00A0"}Rental`,
```

Comment:
Add a comment about the need to use non-breakable spaces explicitly.

Reply:
Added in 1bf58158cfe30c8afdd1a09b73bb9e4649f31c9e